src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,61 @@
+package ui;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+
+    private static final String LOGO = " ____        _        \n"

```

Comment:
I like how you made some of the strings constant! This is something I need to improve on myself.